app/workers/messaging.php

```diff
@@ -47,6 +49,11 @@ public function run(): void
             return;
         }
 
+        if (empty($this->from)) {
+            Console::info('Skipped sms processing. No phone number has been set.');
+            return;
+        }
```

Comment:
This will fail the tests - can you add a phone number to our developer `.env`?

Reply:
okay 👍

Reply:
done